src/hypertrace/agent/config/agent-config.yaml

```diff
@@ -0,0 +1,17 @@
+service_name: pythonagent_001
```

Comment:
Is this file needed? Could you please elaborate on why we need to check in this?

Reply:
This file sets unique configuration values that we are then checking in the unittest for validating that config file values are read correctly.

Reply:
Then I'd rather make it explicit in the name. We can address it in a next PR.